### DISCUSSION TITLE
Welcome to archimmich Discussions!

### DISCUSSION THREAD

**Discussion by osa911** [osa911/archimmich#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sebastianerbs-hub** [comment]
Is there also a possibility to use this function as an upload interface for Immich?

**Reply by osa911** [reply]
No, only export from Immich.
I remember someone tried to create an interface for [immich-go](https://github.com/simulot/immich-go), but I did not follow the progress.

**Reply by sebastianerbs-hub** [reply]
Okay, thank you very much. I consider your work to be very well done. However, I am currently facing the issue that I need to re-upload around 300 directories to Immich.

**Reply by sebastianerbs-hub** [reply]
I will implement and test your suggestion. I believe it will be useful for this use case. Thank you.
